dbtvault-dev/macros/materialisations/vault_insert_by_rank_materialization.sql

```diff
@@ -56,8 +56,6 @@
 
             {{ dbt_utils.log_info("Running for {} {} of {} on column '{}' [{}]".format('rank', iteration_number, min_max_ranks.max_rank, rank_column, model.unique_id)) }}
 
-            {% set tmp_relation = make_temp_relation(this) %}
```

Comment:
Have we taken temp tables out? Can you explain this please?

Reply:
Deleted during testing, but I then forgot to restore it.